Assets/scripts/MenuManager.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    public void StartGame()
+    {
+        SceneManager.LoadScene(2);
+    }
+
+    public void ExitGame()
+    {
+#if UNITY_EDITOR
+        UnityEditor.EditorApplication.isPlaying = false;
+#else
+        PlayExitSound();
```

Comment:
PlayExitSound function doesn't exist. So when we create a build, it will failed.
Plus, I think because we quit the game, we wouldn't hear any sounds.